utils/src/trustBonus.ts

```diff
@@ -31,7 +31,7 @@ export const fetchTrustBonusScore = async (addresses: string[]): Promise<TrustBo
   const url = `https://gitcoin.co/grants/v1/api/trust-bonus?addresses=${addresses.join(',')}`;
 
   try {
-    const response = await fetch(url);
+    const response = await fetch(url, { mode: 'no-cors' });
```

Comment:
Why did we need to add the `no-cors` flag?

Reply:
Ahh we didn't, I was getting CORS errors so I added this to silence them!